pkg/controller/containersource/reconcile.go

```diff
@@ -97,7 +97,7 @@ func (r *reconciler) Reconcile(ctx context.Context, object runtime.Object) (runt
 				r.recorder.Eventf(source, corev1.EventTypeNormal, "DeploymentBlocked", "waiting for %v", err)
 				return object, err
 			}
-			r.recorder.Eventf(source, corev1.EventTypeNormal, "Deployed", "Created deployment %q", deploy.Name)
+			r.recorder.Eventf(source, corev1.EventTypeNormal, "Deployed", "Created deployment %q", args.Name)
```

Comment:
Why not use the actual deployment name here? Won't it be different from `args.Name` (because it's generated)?